python/foglamp/services/core/server.py

```diff
@@ -34,7 +42,7 @@
 # FOGLAMP_ROOT env variable
 _FOGLAMP_ROOT= os.getenv("FOGLAMP_ROOT", default='/usr/local/foglamp')
 _SCRIPTS_DIR= os.path.expanduser(_FOGLAMP_ROOT + '/scripts')
-
+_ENV = os.getenv('FOGLAMP_ENV', 'DEV')
```

Comment:
As decided, we are always in production mode! 💯  no DEV env var (even no test ?!)

Reply:
Yes, this is a copy and paste from similar code is scheduler.
@praveen-garg Are we currently using this variable?

Reply:
only in scheduler for test as Scheduler is in core process and (pytest) test is just another process

Reply:
This must go. Looking at the usage it appears to just bypass getting the storage registration record. The  test harness should make sure a record is in place rather than have a different code path when the test is running.